InvenTree/InvenTree/tasks.py

```diff
@@ -306,4 +307,35 @@ def send_email(subject, body, recipients, from_email=None):
         from_email,
         recipients,
         fail_silently=False,
+        html_message=html_message
     )
+
+
+def notify_low_stock(stock_item):
+    """
+    Notify users who have starred a part when its stock quantity falls below the minimum threshold
+    """
+
+    from allauth.account.models import EmailAddress
+    starred_users = EmailAddress.objects.filter(user__starred_parts__part=stock_item.part)
+
+    if len(starred_users) > 0:
+        logger.info(f"Notify users regarding low stock of {stock_item.part.name}")
+        body = f'Hi, {stock_item.part.name} is low on stock. Kindly do the needful.'
+        context = {
+            'part_name': stock_item.part.name,
+            # Part url can be used to open the page of part in application from the email.
+            # It can be facilitated when the application base url is accessible programmatically.
+            # 'part_url': f'{application_base_url}/part/{stock_item.part.id}',
+
+            'message': body,
+
+            # quantity is in decimal field datatype. Since the same datatype is used in models,
+            # it is not converted to number/integer,
+            'part_quantity': stock_item.quantity,
+            'minimum_quantity': stock_item.part.minimum_stock
+        }
+        subject = f'Attention! {stock_item.part.name} is low on stock'
```

Comment:
This needs to be translated

Reply:
should something be done from my side?

Reply:
If you know how django handles it (its the _ function) yes, ping me if I should do it

Reply:
I shall do a little research and see if I can do it.

Reply:
Refer to the django translation docs - https://docs.djangoproject.com/en/3.2/topics/i18n/translation/

- In python code we use the ugettext_lazy function
- In template files we use `{% trans %}`